libs/shared/utils/src/lib/shouldLinkBeAnAnchorTag.ts

```diff
@@ -0,0 +1,11 @@
+import { ProjectBasePath } from '@island.is/shared/constants'
+
+/** Some links point users to another project (like if a user clicks on a /minarsidur link in the web project),
+ * then the link should be an anchor tag instead of a nextlink for example
+ * */
+export const shouldLinkBeAnAnchorTag = (path: string) => {
+  for (const basePath of Object.values(ProjectBasePath)) {
+    if (path.includes(basePath)) return true
+  }
+  return false
+}
```

Comment:
?
```suggestion
export const shouldLinkBeAnAnchorTag = (path: string) => Object.values(ProjectBasePath).includes(path)
```

Reply:
Let's say the path is "/minarsidur/eitthvad"

If that gets passed in it'll return true with the previous version of the function but if we use your version of the function it'll return false.

Reply:
Yup agreed. You version handles nested routes which we need to do.